tests/integration/security/authorization_test.go

```diff
@@ -1157,7 +1172,10 @@ func TestAuthorization_Path(t *testing.T) {
 			for _, a := range []echo.Instances{a, vm} {
 				for _, srcCluster := range t.Clusters() {
 					t.NewSubTest(fmt.Sprintf("In %s", srcCluster.StableName())).Run(func(t framework.TestContext) {
-						b := apps.B.GetOrFail(t, echo.InCluster(srcCluster).And(echo.Namespace(ns.Name())))
+						b := apps.B.Match(echo.InCluster(srcCluster).And(echo.Namespace(ns.Name())))
+						if !util.CheckExistence(b) {
```

Comment:
nit: you can call t.Skip in CheckExistence to avoid 2 extra lines on each call?

could even make it return `b` so you can use it inline

Reply:
Good point! Will revise it later.

Reply:
Updated CheckExistence to skip the test if any workload is not available.